### ISSUE TITLE
Not working since ios 13.3

### ISSUE THREAD

**Issue by sanctus671** [samkelleher/cordova-plugin-applepay#56]
Hey there,

I'm getting an issue when calling makePaymentRequest since the latest iOS update to 13.3. canMakePayments works fine, but when I call the makePaymentRequest function, the app crashes. Was working fine prior to 13.3 so could be something related to the update. Any ideas?

**Comment by skibaalex** [comment]
Im looking into integrating apple pay and on the documentation it says that it compatible up to ios 12 did you manage to fix the problem? @sanctus671

**Comment by sanctus671** [comment]
Unfortunately not. I couldn't find any alternatives either. Most plugins for Apple Pay in Cordova don't seem to be well maintained. Fingers crossed there is an update soon. If you manage to get it working in iOS > 13.3, let me know.

**Comment by ghost** [comment]
Could you post the error that you get when the app crashes?

**Comment by myk-lop** [comment]
Hi there) Any updates on this issue guys?

**Comment by madmacc** [comment]
Any plans to update this plugin to be compatible with iOS 13-14.5?

**Comment by sdbarrington** [comment]
Any updates does this plugin still work with latest iOS? only other one seems to be ionic's own which they charge an arm and a leg for...

**Comment by sdbarrington** [comment]
Oh wow, literally just discovered this plugin in another discussion, looks to solve everyones problems: https://www.npmjs.com/package/@capacitor-community/stripe

**Comment by madmacc** [comment]
@sdbarrington There is a working one for Capacitor. https://github.com/capacitor-community/stripehttps://github.com/capacitor-community/stripe

**Comment by madmacc** [comment]
Jinx

**Comment by sdbarrington** [comment]
yes I saw that! (finally). think they need to promote it higher and stop promoting this one as the 'Apple Pay' option in the docs. Will looks t integrating it, thanks for your help!

**Comment by madmacc** [comment]
> yes I saw that! (finally). think they need to promote it higher and stop promoting this one as the 'Apple Pay' option in the docs. Will looks t integrating it, thanks for your help!

Yes definitely.
This one is pretty new I was helping to test it a bit last month.
Seems to work well though.

**Comment by sdbarrington** [comment]
Yes only issue is it’s capacitor 3 only. Which still has some issues with other plug-in.

Scott Barrington
----------------

Director

Avon River

+1 415 283 9441 ( tel:+1+415+283+9441 ) | +64 27 296 2047 ( tel:+64+27+296+2047 )

***@***.***

www.avon-river.com ( https://www.avon-river.com/ )

twitter ( https://twitter.com/SDBarrington )

linkedin ( https://www.linkedin.com/in/barrington/ )

This Email Is Not An Acceptable Offer And Does Not Evidence Any Intention By The Sender To Enter Into A Contract. This email and any attachments to it may be confidential and are intended solely for the use of the individual to whom it is addressed. Any views or opinions expressed are solely those of the author and do not necessarily represent those of Avon River Limited.
If you are not the intended recipient of this email, you must neither take any action based upon its contents, nor copy or show it to anyone.
Please contact the sender if you believe you have received this email in error.
